src/node/executor.cpp

```diff
@@ -69,6 +69,12 @@ void Executor::execute(std::shared_ptr<PbftBlock> blk) {
   cv_.notify_one();
 }
 
+void Executor::executeSynced(const std::shared_ptr<PbftBlock> &blk) {
+  assert(final_chain_->last_block_number() < blk->getPeriod());
```

Comment:
The PBFT block is going to execute should be exactly in order.
I think should be assert(final_chain_->last_block_number() + 1 == blk->getPeriod())

Reply:
agree with @gaoqi8649

Reply:
so this one is invalid too right ? 
https://github.com/Taraxa-project/taraxa-node/blob/develop/src/node/executor.cpp#L61

Reply:
No, that one is correct. 
Your new executeSynced is synchronous, that will execute PBFT block one by one in exact ordering
The old execute function is asynchronous, that will execute a bunch of PBFT blocks

Reply:
I think after your changes, you are right. Since PBFT only will get multiple blocks from syncing. Each time only be able to vote one block.
So the PR will change back PBFT blocks execution in synchronous.
@justinsnapp I think we need to discuss on that if it's OK to change back on synchronous execution

Reply:
@JakubFornadel @justinsnapp How about move synced PBFT blocks execution to taraxa capability thread?